config/dynamic-scaling-configuration.yaml

```diff
@@ -1,7 +1,17 @@
+#This configuration file contains information about each Kafka instance types, dynamic scaling configuration
+#
+#The following properties must be defined for each Kafka instance type:
+#   - capacity_reservation: the number of reservation to be made on each data plane cluster
+#
+#The following properties must be defined for each instance types compute node configuration:
+#   - max_compute_nodes: the maximum number of nodes needed to create an appropriate machine pool with the configuration needed by the autoscaler
+
 ---
 developer:
+  capacity_reservation: 3 
```

Comment:
When reviewing this it didn't seem clear to me what `capacity_reservation` is. Nodes? kafka instances? pause pods?

Maybe that's an indicative of needing a more specific name?

cc @VanillaSpoon @machi1990

Reply:
Good point. `reserved_streaming_units` or `reserved_kafka_streaming_units`?

Reply:
Streaming units are limited to the kafka domain (at least for now) so both seem correct to me.

Reply:
/cc @JasonPowr see the above suggestions for the naming of the field.